pkg/build/create.go

```diff
@@ -14,13 +14,19 @@ import (
 )
 
 type CreateBuild struct {
-	BuilderName  string            `help:"The name of the entity that triggered the build. Could be a user, system etc."`
-	Metadata     map[string]string `help:"Additional build information"`
-	ReleaseStage string            `help:"The release stage (eg, production, staging) that is being released (if applicable)."`
-	Provider     string            `help:"The name of the source control provider that contains the source code for the build."`
-	Repository   string            `help:"The URL of the repository containing the source code being deployed."`
-	Revision     string            `help:"The source control SHA-1 hash for the code that has been built (short or long hash)"`
-	Path         utils.UploadPaths `arg:"" name:"path" help:"Path to the project directory" type:"path" default:"."`
+	BuilderName      string            `help:"The name of the entity that triggered the build. Could be a user, system etc."`
+	Metadata         map[string]string `help:"Additional build information"`
+	ReleaseStage     string            `help:"The release stage (eg, production, staging) that is being released (if applicable)."`
+	Provider         string            `help:"The name of the source control provider that contains the source code for the build."`
+	Repository       string            `help:"The URL of the repository containing the source code being deployed."`
+	Revision         string            `help:"The source control SHA-1 hash for the code that has been built (short or long hash)"`
+	Path             utils.UploadPaths `arg:"" name:"path" help:"Path to the project directory" type:"path" default:"."`
+	Version          string            `help:"The version of the application." xor:"app-version,version"`
+	AppVersion       string            `help:"(deprecated) The version of the application." xor:"app-version,version"`
```

Comment:
We call this `VersionName` on Android - is it worth always using this term in the CLI? Especially to avoid a `--version` switch which usually has a different meaning.